modules/test/playwright/tests/export-import-web/pages/ExportImportPage.ts

```diff
@@ -120,7 +120,15 @@ export class ExportImportPage {
 	async downloadExportProcess(name: string) {
 		const downloadPromise = this.page.waitForEvent('download');
 
-		await this.page.locator('//*[contains(@href, "' + name + '")]').click();
+		await this.page
+			.locator(
```

Comment:
Maybe its late but I think this could be simplified :)
```
 .locator(
                '//h2[span[normalize-space()="' +
                    name +
                    '"]]/span/a'
            )
```

Reply:
@albertojml Could you use the suggestion and forward the PR?